OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -78,18 +78,29 @@ public override bool Tick(Actor self)
 				if (aircraft != null)
```

Comment:
The entire block handling reservation, takeoff and rallypoints is now repeated twice, so it would make sense to pull this into a method so as to not repeat ourselves.